oozie-to-airflow/templates/workflow.tpl

```diff
@@ -14,8 +14,18 @@
   limitations under the License.
  #}
 
+{%- for depedency in dependencies %}
+{{ depedency }}
```

Comment:
```suggestion
{{ dependency }}
```

Reply:
I updated the PR. Thanks a lot. You're awesome.